remoteworkitem/trackerquery_repository_test.go

```diff
@@ -81,20 +91,26 @@ func (test *TestTrackerQueryRepository) TestExistsTrackerQuery() {
 		err := test.trackerRepo.Create(ctx, &tracker)
 		require.NoError(t, err)
 
-		query, err := test.queryRepo.Create(ctx, "abc", "xyz", tracker.ID, space.SystemSpace)
-		require.NoError(t, err)
+		query := remoteworkitem.TrackerQuery{
+			Query:     "abc",
+			Schedule:  "xyz",
+			TrackerID: tracker.ID,
+			SpaceID:   testFxt.Spaces[0].ID,
+		}
+		err = test.queryRepo.Create(ctx, &query)
+		assert.Nil(t, err)
```

Comment:
Please use the already well established `require.NoError` here and search for any further usage.
```suggestion
	require.NoError(t, err)
```

Reply:
corrected here 4cee394

Reply:
Please see my comment on that revision https://github.com/fabric8-services/fabric8-wit/commit/4cee3948982fde26b55c6b0c2d82caa538e15c94#r31463970

Reply:
Already done this. https://github.com/fabric8-services/fabric8-wit/pull/2351#issuecomment-442332827